src/api/restApi.ts

```diff
@@ -80,6 +81,23 @@ export class RestApi {
         .json();
   }
 
+  // tslint:disable-next-line:variable-name
+  private _postRes(wretcher: () => Wretcher) {
```

Comment:
What does Res means?

Reply:
It means that we expect raw response: https://github.com/elbywan/wretch#rescb-response--response--any